packages/client/hooks/useSpotlightResults.ts

```diff
@@ -8,8 +8,8 @@ import graphql from 'babel-plugin-relay/macro'
 type ResultGroups = useSpotlightResultsLocalQueryResponse['viewer']['similarReflectionGroups']
 
 const useSpotlightResults = (
-  spotlightGroupId: string | undefined,
-  searchQuery: string | undefined,
+  spotlightGroupId?: string,
+  searchQuery?: string | null,
```

Comment:
Adding `searchQuery` in this PR meant that all the parent components had to add that to their fragments.
Since `spotlightGroupId, searchQuery` both come from the relay store we could replace them with a `meetingRef` which means the parent components don't have to fetch data on behalf of this hook.

for example

```ts
const useSpotlightResults = (meetingRef) => {
  const meeting = readInlineData<useSpotlightResults_meeting$key>(
    graphql`
      fragment useSpotlightResults_meeting on RetrospectiveMeeting @inline {
        spotlightGroup {
          id
        }
        spotlightSearchQuery
      }
    `,
    meetingRef
  )
  const {spotlightGroup, spotlightSearchQuery = ''} = meeting
  const spotlightGroupId = spotlightGroup?.id ?? ''
```

PS I think we could also remove the `getReflections` param since this query is `store-only`, but maybe I'm missing something?

Reply:
Ah nice, I like that idea! Update pushed 👍